### PR TITLE
CRM: Adding more default and global placeholder information

### PR DESCRIPTION
**Pull request by coder-karen**
## Proposed changes:

* This PR focuses on fixing a few placeholders (and groups of placeholders) that ideally should be showing by default.
* In particular, those are `##BIZ-EXTRA-INFO##`, `##BIZ-INFO##`, `##SOCIAL-LINKS##`, and `##QUOTE-NOTES##` ( as it looks like this should be default ), as well as Quote custom fields.
* Please see the comment here - https://github.com/Automattic/zero-bs-crm/issues/1632#issuecomment-1461670297 - for more context around changes and what else may need to happen. One follow up will be secondary address fields for example (now here - https://github.com/Automattic/jetpack/pull/29396 ). And another follow up may be clarifying where certain placeholders are set up to work.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/1632

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this, using the Jetpack Beta plugin on a test site with this branch applied or testing locally with the branch checked out, test adding the `##SOCIAL-LINKS##` placeholder in a quote template (with social links having been filled in here: `wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=bizinfo`). 
* Other placeholders to test include `##QUOTE-NOTES##` with a quote note added to the quote you are adding a quote template to. And also `##BIZ-INFO##`, having filled out information here - `/wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=bizinfo`, and `##BIZ-EXTRA-INFO##` having filled out the 'Extra Invoice Info' on the invoices settings page: wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=invbuilder

### CODE REVIEW

**Comment by cleacos** [comment]
I've tried to test this PR, but I could not... I created a quote with those placeholders, and it doesn't work, I see the placeholders in the output PDF.

File: projects/plugins/crm/includes/ZeroBSCRM.AJAX.php [hunk 1]
```diff
@@ -1345,6 +1345,13 @@ function ZeroBSCRM_get_quote_template() {
 				$quote_date = date( 'd/m/Y', time() );
 			}
 
+			if ( empty( $quote_notes ) ) {
+				$quote_notes = '';
```
Comment (cleacos) [inline projects/plugins/crm/includes/ZeroBSCRM.AJAX.php]: Two things here... 

If we are checking that `$quote_notes` is empty, do we need to ensure/force it set it as an empty string after the condition?

`$quote_notes = '0';` >> `empty( $quote_notes )` >> `true`

**Review by cleacos — DISMISSED** [review]
It works well and as expected. I left a little question in the code, but it's not a blocker.

LGTM!

File: projects/plugins/crm/includes/ZeroBSCRM.AJAX.php (hunk 1 above)
Comment (coder-karen) [reply projects/plugins/crm/includes/ZeroBSCRM.AJAX.php]: You're right, it doesn't look like we need to do that actually, thanks! I originally had the empty string as the 'else', then moved it as the negative case to before the if statement, but even though the variable is undefined vs an empty string it all still works. Just added another commit.

**Review by cleacos — APPROVED** [review]
LGTM!

**Pull request merged**